yarn.lock

```diff
@@ -14382,20 +14382,20 @@ typeforce@^1.11.3, typeforce@^1.11.5:
   resolved "https://registry.yarnpkg.com/typeforce/-/typeforce-1.18.0.tgz#d7416a2c5845e085034d70fcc5b6cc4a90edbfdc"
   integrity sha512-7uc1O8h1M1g0rArakJdf0uLRSSgFcYexrVoKo+bzJd32gd4gDy2L/Z+8/FjPnU9ydY3pEnVPtr9FyscYY60K1g==
 
+typescript@4.5.5:
+  version "4.5.5"
+  resolved "https://registry.yarnpkg.com/typescript/-/typescript-4.5.5.tgz#d8c953832d28924a9e3d37c73d729c846c5896f3"
+  integrity sha512-TCTIul70LyWe6IJWT8QSYeA54WQe8EjQFU4wY52Fasj5UKx88LNYKCgBEHcOMOrFF1rKGbD8v/xcNWVUq9SymA==
+
 typescript@^3.6.4:
   version "3.9.10"
   resolved "https://registry.yarnpkg.com/typescript/-/typescript-3.9.10.tgz#70f3910ac7a51ed6bef79da7800690b19bf778b8"
   integrity sha512-w6fIxVE/H1PkLKcCPsFqKE7Kv7QUwhU8qQY2MueZXWx5cPZdwFupLgKK3vntcK98BtNHZtAF4LA/yl2a7k8R6Q==
 
 typescript@^4.5.4:
-  version "4.5.5"
-  resolved "https://registry.yarnpkg.com/typescript/-/typescript-4.5.5.tgz#d8c953832d28924a9e3d37c73d729c846c5896f3"
-  integrity sha512-TCTIul70LyWe6IJWT8QSYeA54WQe8EjQFU4wY52Fasj5UKx88LNYKCgBEHcOMOrFF1rKGbD8v/xcNWVUq9SymA==
-
-typescript@~4.2.4:
-  version "4.2.4"
-  resolved "https://registry.yarnpkg.com/typescript/-/typescript-4.2.4.tgz#8610b59747de028fda898a8aef0e103f156d0961"
-  integrity sha512-V+evlYHZnQkaz8TRBuxTA92yZBPotr5H+WhQ7bD3hZUndx5tGOa1fuCgeSjxAzM1RiN5IzvadIXTVefuuwZCRg==
+  version "4.6.2"
```

Comment:
can we remove the explicit `typescript` dep from our internal packages? It should only exist in the root package.json:

```
$ yarn why typescript
yarn why v1.22.17
[1/4] 🤔  Why do we have the module "typescript"...?
[2/4] 🚚  Initialising dependency graph...
[3/4] 🔍  Finding dependency...
[4/4] 🚡  Calculating file sizes...
=> Found "typescript@4.5.5"
info Has been hoisted to "typescript"
info Reasons this module exists
   - "workspace-aggregator-846fbfd9-bfaa-4869-ba14-497b73770ea7" depends on it
   - Specified in "devDependencies"
   - Hoisted from "_project_#typescript"
info Disk size without dependencies: "61.75MB"
info Disk size with unique dependencies: "61.75MB"
info Disk size with transitive dependencies: "61.75MB"
info Number of shared dependencies: 0
=> Found "@bitgo/blockapis#typescript@4.6.2"
info This module exists because "_project_#@bitgo#blockapis" depends on it.
info Disk size without dependencies: "62.45MB"
info Disk size with unique dependencies: "62.45MB"
info Disk size with transitive dependencies: "62.45MB"
info Number of shared dependencies: 0
=> Found "@bitgo/utxo-bin#typescript@4.6.2"
info This module exists because "_project_#@bitgo#utxo-bin" depends on it.
info Disk size without dependencies: "62.45MB"
info Disk size with unique dependencies: "62.45MB"
info Disk size with transitive dependencies: "62.45MB"
info Number of shared dependencies: 0
=> Found "bls12377js#typescript@3.9.10"
info This module exists because "_project_#@bitgo#account-lib#@celo#contractkit#@celo#utils#bls12377js" depends on it.
info Disk size without dependencies: "52.33MB"
info Disk size with unique dependencies: "52.33MB"
info Disk size with transitive dependencies: "52.33MB"
info Number of shared dependencies: 0
✨  Done in 1.06s.
```

We'll have to do something else about the really old version coming in from `@celo/contractkit` separately